pkg/manager/meta/reclaim_policy_manager.go

```diff
@@ -50,6 +51,11 @@ func (m *reclaimPolicyManager) SyncDM(dc *v1alpha1.DMCluster) error {
 }
 
 func (m *reclaimPolicyManager) sync(kind string, obj runtime.Object, isPVReclaimEnabled bool, policy corev1.PersistentVolumeReclaimPolicy) error {
+	if m.deps.PVLister == nil {
+		klog.V(4).Infof("persistent volumes lister is unavailable, skip syncing reclaim policy for %s. this may be caused by no relevant permissions", kind)
```

Comment:
```suggestion
		klog.V(4).Infof("Persistent volumes lister is unavailable, skip syncing reclaim policy for %s. This may be caused by no relevant permissions", kind)
```